public/app/plugins/panel/geomap/style/markers.ts

```diff
@@ -121,13 +121,14 @@ const makers: SymbolMaker[] = [
     aliasIds: [MarkerShapePath.square],
     make: (cfg: StyleConfigValues) => {
       const radius = cfg.size ?? DEFAULT_SIZE;
+      const rotation = cfg.rotation ?? 0;
       return new Style({
         image: new RegularShape({
           stroke: new Stroke({ color: cfg.color, width: cfg.lineWidth ?? 1 }),
           fill: getFillColor(cfg),
           points: 4,
           radius,
-          angle: Math.PI / 4,
+          rotation: (rotation * Math.PI) / 180 + Math.PI / 4,
```

Comment:
I may not be seeing something, but why the variation on calculating rotation for the different shapes?

I'm seeing two different versions:
`(rotation * Math.PI) / 180 + Math.PI / 4`
`(rotation * Math.PI) / 180`